src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinPlayerEntity.java

```diff
@@ -270,4 +273,15 @@ private void hookPlaySound5(Entity target, CallbackInfo ci) {
         }
     }
 
+    /**
+     * Sneak height fix
+     * @see net.ccbluex.liquidbounce.features.module.modules.movement.ModuleNoPose
+     */
+    @Inject(method = "getBaseDimensions", at = @At("RETURN"), cancellable = true)
+    private void hookGetBaseDimensions(EntityPose pose, CallbackInfoReturnable<EntityDimensions> cir) {
+        if (pose == EntityPose.CROUCHING) {
+            EntityDimensions dimensions = ModuleNoPose.INSTANCE.modifySneakHeight();  /* If module/setting is not enabled, modifySneakHeight() returns null */
```

Comment:
:boring: unnecessary comment

```suggestion
            EntityDimensions dimensions = ModuleNoPose.INSTANCE.modifySneakHeight();
```

Reply:
`@Nullable EntityDimensions dimensions = ...`